docs/source/config.json

```diff
@@ -36,19 +36,21 @@
       "Security": {
         "CORS": "/configuration/cors",
         "CSRF prevention": "/configuration/csrf",
-        "JWT Authentication": [
-          "/configuration/authn-jwt",
-          [
-            "enterprise"
-          ]
-        ],
+        "Access control": {
```

Comment:
I thought it might be better to group the authentication/authorization docs in a subheader, but still need a better title for it. ("Access control" could apply to a few other of these pages...)